packages/core/src/RenderingEngine/VolumeViewport.ts

```diff
@@ -668,10 +680,11 @@ class VolumeViewport extends BaseVolumeViewport {
       this.updateClippingPlanesForActors(this.getCamera());
     }
 
-    const imageVolume = cache.getVolume(volumeActor.uid);
+    volumeId = volumeId ?? this.getVolumeId();
```

Comment:
I think volumeId ||= this.getVolumeId() is probably better - it disallows null/empty string as the volume id (and also 0), which I think is best.  It also only re-assigns if the current value is empty, whereas your statement always re-assigns.